skfem/element/element_hex/element_hex_s2.py

```diff
@@ -0,0 +1,87 @@
+import numpy as np
+
+from ..element_h1 import ElementH1
+from ...mesh.mesh3d import MeshHex
+
+
+class ElementHexS2(ElementH1):
```

Comment:
`ElementQuadS2` is derived from `ElementQuad2` in which case for consistency `ElementHexS2` should be derived from `ElementHex2`, but perhaps it shouldn't have?   I've sometimes read that object-oriented inheritance is supposed to mean 'is a' but don't know how valid or important that is.  Anyway I think the tree should be consistent between quadrilaterals and hexahedra here.

Reply:
I changed how `ElementQuadS2` gets defined.